build/templates/_converters.py.mako

```diff
@@ -254,6 +254,16 @@ def _convert_to_bytes(value):  # noqa: F811
     pass
 
 
+<%
+# This converter is only needed by nidigital
```

Comment:
This is true today. That said, the converter is generic and as a matter of fact I bet there are places in `niswitch` we could/should have done this.

I'm ok not hard-coding this driver specific logic here but I'm ok if you or Mark push back.

Reply:
That's true, I'll make it available to all drivers. Do you know where in `niswitch` this could be used? I'll create a tech-debt issue for it.

Reply:
Well unfortunately that would be a source breaker :(
We talked about opening issues for a 2.0 source breaker milestone. Perhaps this can go there.

Reply:
Created a `2.0 Source Breakers` milestone and opened an issue against it.

https://github.com/ni/nimi-python/milestone/29